docs/appendices/release-notes/unreleased.rst

```diff
@@ -241,4 +241,15 @@ Fixes
   and/or broken replica shards when ingesting into a table using a number type
   column with explicit turned off indexing by declaring ``INDEX OFF``.
 
+- Fixed an issue which caused ``INSERT INTO... SELECT...`` statements to fail
+  on insertion of a correct value into partitioned column of a none-string type
+  with a ``CHECK`` constraint.
+
+- Fixed an issue which caused ``INSERT INTO... SELECT...`` statements to fail
+  if source table has a generated column of a none-string type, computed basing
+  on partitioned by column.
```

Comment:
```suggestion
  if the source table has a generated column derived from a partitioned column that's not a string.
```

Not sure if I understood it correctly?

Reply:
I will rephrase, it's basically about 

 case 

`CREATE TABLE(... generated_regular_from_parted INT as parted+1) PARTITIONED BY (parted)`
causing record insertion failure because of casting.

Smth like 
"Fixed an issue which prevented INSERT INTO ... SELECT .. from inserting any records if the source table had a generated column referring to a partitioned column of none-string type."

Reply:
Why does the _source_ table matter here? The fixes are on the target table / indexer side?

Reply:
oh, yep, would be "target"